src/main/java/net/mcreator/element/types/Plant.java

```diff
@@ -139,8 +142,12 @@ public Plant(ModElement element) {
 		this.jumpFactor = 1.0;
 		this.speedFactor = 1.0;
 
+		this.suspiciousStewEffect = "SPEED";

```

Comment:
So far default was saturation so I think we should keep this the same, same with duration, which was 0 so far